pages/common/mosh.md

```diff
@@ -0,0 +1,33 @@
+# mosh
+
+> Mobile Shell (`mosh`) is remote terminal application that allows roaming.
+> `mosh` is a more robust and responsive replacement for SSH.
+> Visit <https://mosh.org> for more information.
+
+- Typical usage:
```

Comment:
I'm not sure I was clear. I meant that "Typical usage" does not really fit the format of descriptions we use in the project. We try to have every example stand alone, rather than depending on previous content of the page. Does that make sense to you?